q2_dada2/plugin_setup.py

```diff
@@ -71,14 +71,15 @@
         'n_reads_learn': ('The number of reads to use when training the '
                           'error model. Smaller numbers will result in a '
                           'shorter run time but a less reliable error '
-                          'model.'),
+                          'model. If 0 is provided, all input reads will be '
+                          'used.'),
```

Comment:
I'm removing the description of the `0` behavior, since @benjjneb suggests that we wouldn't want to do this for a MiSeq run (which is most common for us). We're then going to require this value to be greater than zero in a separate PR where we include checks of these values.